Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/hitscan.yml

```diff
@@ -75,12 +90,13 @@
   impactFlash:
     sprite: Objects/Weapons/Guns/Projectiles/projectiles.rsi
     state: impact_xray
+  collisionMask: CollisionGroup.MobImpassable
```

Comment:
linter fails because its trying to read this as part of damage: which is a damageSpecifier, move this  before that and it should not complain anymore

Reply:
Wish I had seen this before, thanks